gadgets/snapshot_process/test/integration/snapshot_process_test.go

```diff
@@ -33,7 +32,7 @@ import (
 
 type snapshotProcessEntry struct {
 	eventtypes.CommonData
```

Comment:
What do you think about also adding an alias for this to use only the utils package from the tests?

Reply:
Yup, we can do that too.
I'll look into it

Reply:
Yup, I have done this too